### DISCUSSION TITLE
James Perkins - Clerk, Unkey

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#198] [Announcements]
- https://twitter.com/james_r_perkins
- https://clerk.com/
- https://unkey.dev/devtools.fm

## Intro

Our guest today is James Perkins. James is the the co-found of unkey, an open source API key management solution. James also previously worked at clerk, a popular user management service. James, it's great to have you on! Before we start, is there anything else you'd like to tell our listeners about yourself?

## Questions

- [x] What is clerk and how did you contribute to it?
- [x] What is unkey and why did you build it? Were you inspired by clerk?
- [x] Building a business as a side project seems like a whole lot of work? What tech are you using to accelerate that?
- [x] You recently left clerk to full time on unkey. How hard was that decision to make? What was the tipping point?
- [x] Like clerk you have many sdks (front ends) to use unkey. What platforms do you support and what the integration you're most proud of?
- [x] What feature do you think makes unkey stand out? What type of workflows are made easier?
- [x] How secure is it?
- [x] What steps have you taken to make it fast?
- [x] The service is young where do you want to take it? 

### Future

- [x] What's your spiciest dev toke?
- [x] What do you think the future of backend is? more managed composable services?

**Comment by hipstersmoothie** [comment]
https://github.com/intel/Wasmnizer-ts
https://carton.run/

**Comment by just-be-dev** [comment]
Tooltips: 
- https://github.com/laurentpayot/verticalize
- https://deno.com/blog/queues

**Comment by perkinsjr** [comment]
Tooltips: 

https://reflect.app/
Rise Cal : https://www.risecalendar.com/
Hanko: https://www.hanko.io/
